src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -332,6 +332,15 @@ private static bool IsFloatPointExpressionOfUnknownPrecision(BoundExpression rew
                         explicitCastInCode: explicitCastInCode,
                         rewrittenType: (NamedTypeSymbol)rewrittenType);
 
+                case ConversionKind.MethodGroup:
+                    {
+                        // we eliminate the method group conversion entirely from the bound nodes following local lowering
+                        var mg = (BoundMethodGroup)rewrittenOperand;
+                        var method = oldNode.SymbolOpt;
+                        Debug.Assert((object)method != null);
+                        var receiver = (method.IsStatic && !oldNode.IsExtensionMethod) ? _factory.Type(method.ContainingType) : VisitExpression(mg.ReceiverOpt);
```

Comment:
>_factory.Type(method.ContainingType) [](start = 89, length = 36)

Consider adjusting _factory.Syntax before creating synthetic node so that it would be easier to investigate any issues related to that node and avoid confusing a fellow developer with a "random" syntax associated with the node.